src/tasks/compute-merkle-root.ts

```diff
@@ -0,0 +1,24 @@
+import { task } from "hardhat/config";
+
+import { computeProofs, parseCsvFile } from "../ts";
+import { Args } from "../ts/lib/common-interfaces";
```

Comment:
We probably shouldn't use this `Args` here as it also contains `settings`, so if we ever come around to validating the script input against `Args`, this would fail. It could just be:
```
interface Args {
  claims: string,
}
```